papermill_origami/noteable_dagstermill/ops.py

```diff
@@ -289,116 +290,9 @@ def _t_fn(step_context, inputs):
-        tags (Optional[Dict[str, str]]): If set, additional tags used to annotate solid.
-            Dagster uses the tag keys `notebook_path` and `kind`, which cannot be
-            overwritten by the user.
-    Returns:
-        :py:class:`~dagster.SolidDefinition`
-    """
-    check.str_param(name, "name")
-    check.str_param(notebook_path, "notebook_path")
-    input_defs = check.opt_list_param(input_defs, "input_defs", of_type=InputDefinition)
-    output_defs = check.opt_list_param(output_defs, "output_defs", of_type=OutputDefinition)
-    required_resource_keys = set(
-        check.opt_set_param(required_resource_keys, "required_resource_keys", of_type=str)
-    )
-
-    extra_output_defs = []
-    if output_notebook_name is not None:
-        required_resource_keys.add("output_notebook_io_manager")
-        extra_output_defs.append(
-            OutputDefinition(name=output_notebook_name, io_manager_key="output_notebook_io_manager")
-        )
-    # backcompact
-    if output_notebook is not None:
-        rename_warning(
-            new_name="output_notebook_name",
-            old_name="output_notebook",
-            breaking_version="0.14.0",
-        )
-        required_resource_keys.add("file_manager")
-        extra_output_defs.append(OutputDefinition(dagster_type=FileHandle, name=output_notebook))
-
-    if isinstance(asset_key_prefix, str):
-        asset_key_prefix = [asset_key_prefix]
-
-    asset_key_prefix = check.opt_list_param(asset_key_prefix, "asset_key_prefix", of_type=str)
-
-    default_description = f"This solid is backed by the notebook at {notebook_path}"
-    description = check.opt_str_param(description, "description", default=default_description)
-
-    user_tags = validate_tags(tags)
-    if tags is not None:
-        check.invariant(
-            "notebook_path" not in tags,
-            "user-defined solid tags contains the `notebook_path` key, but the `notebook_path` key is reserved for use by Dagster",  # noqa: E501
-        )
-        check.invariant(
-            "kind" not in tags,
-            "user-defined solid tags contains the `kind` key, but the `kind` key is reserved for use by Dagster",
-        )
-    default_tags = {"notebook_path": notebook_path, "kind": "ipynb"}
-
-    return SolidDefinition(
-        name=name,
-        input_defs=input_defs,
-        compute_fn=_dm_compute(
-            "define_dagstermill_solid",
-            name,
-            notebook_path,
-            output_notebook_name,
-            asset_key_prefix=asset_key_prefix,
-            output_notebook=output_notebook,  # backcompact
-        ),
-        output_defs=output_defs + extra_output_defs,
-        config_schema=config_schema,
-        required_resource_keys=required_resource_keys,
-        description=description,
-        tags={**user_tags, **default_tags},
-    )
-
-
-def define_noteable_dagstermill_op(
+def define_noteable_dagster_op(
```

Comment:
suggesting this name change to make it slightly shorter and more clear that it's a dagster integration (the name dagstermill sometimes confuses our users, so it might be better to avoid it here if you're up for it)